packages/moonstone/styles/colors-light.less

```diff
@@ -184,6 +184,9 @@
 
 // Spinner
 // ---------------------------------------
+@moon-spinner-color: @moon-darker-gray;
+@moon-spinner-head-color: @moon-spinner-color;
+@moon-spinner-empty-color: fade(@moon-dark-gray, 20%);
 @moon-spinner-bg-color: @moon-button-bg-color;
```

Comment:
I didn't have a spec to follow here, so I just made a "best guess". The background can be turned off, which is why I thought a black ring may be passable/useful. I'm not married to it and am open to coercion.

Reply:
`#979797` (as compared to the current `#686868`)would maintain a similar contrast between foreground and background so that'd be my suggestion if we wanted to change it.